tools/build/rules/android.bzl

```diff
@@ -135,3 +135,30 @@ ndk_vk_validation_layer = repository_rule(
         "ANDROID_NDK_HOME",
     ],
 )
+
+# Enforce NDK version by checking $ANDROID_NDK_HOME/source.properties
+def _ndk_version_check(repository_ctx):
+    # This should be updated in sync with BUILDING.md documentation
+    version = "21.0.6113669"
```

Comment:
Should we have a specific version or should we accept anything starts with 26? Do we want to reject if there is a minor version difference?

Reply:
I guess you meant "that starts with 21" ?

I think we should impose specific versions. There are very few NDK versions made public, for instance in the NDK r21 series, there are only 2 public releases so far:
- `r21` with version `21.0.6113669` (used here)
- `r21d` with version `21.3.6528147`, freshly released a few days ago (updating AGI to use this one is tracked by b/158730767)

And I suspect the next NDK release will be `r22`, around October.

Because our users are unlikely to have many different minor versions of the NDK lying around, I think we should match a precise version.

Reply:
Seems fine for me.